spotinst/resource_spotinst_ocean_spark_virtual_node_group_test.go

```diff
@@ -0,0 +1,185 @@
+package spotinst
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/spotinst/spotinst-sdk-go/service/ocean/spark"
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+
+	"github.com/spotinst/terraform-provider-spotinst/spotinst/commons"
+)
+
+const (
+	testOceanSparkClusterID = "osc-d552c5b5"
+	testOceanDedicatedVngID = "ols-b66e444f"
+)
+
+var oceanSparkClusterID = getOceanSparkClusterID() // NOTE: This needs to be an existing ocean cluster
+var oceanSparkVngID = getOceanSparkVngID()         // NOTE: This needs to be an existing ocean VNG
+
+func init() {
+	resource.AddTestSweepers("spotinst_ocean_spark_virtual_node_group", &resource.Sweeper{
+		Name: "spotinst_ocean_spark_virtual_node_group",
+		F:    testSweepOceanSparkVng,
+	})
+}
+
+func getOceanSparkClusterID() string {
+	// Prefer environment variable
+	oceanSparkClusterID := os.Getenv("TEST_ACC_OCEAN_SPARK_CLUSTER_ID")
+	if oceanSparkClusterID == "" {
+		// Default to hardcoded ID
+		oceanSparkClusterID = testOceanSparkClusterID
+	}
+
+	return oceanSparkClusterID
+}
+
+func getOceanSparkVngID() string {
+	// Prefer environment variable
+	oceanDedicatedVngID := os.Getenv("TEST_ACC_OCEAN_DEDICATED_VNG_ID")
```

Comment:
name this env variable TEST_ACC_OCEAN_SPARK_DEDICATED_VNG_ID? so it is clearer that this is for ocean spark, not ocean